src/lib/Sympa/List.pm

```diff
@@ -3126,6 +3126,11 @@ sub add_list_member {
                 $u->{email});
             next;
         }
+        if ($who eq $self->get_id) {
+            $log->syslog('err', 'Ignoring %s which is the address of the list',
+                $who);
+            next;
+        }
```

Comment:
I feel the codes for error handling in `add.pm` & `subscribe.pm` are redundant.

We'd be better to insert
``` perl
push @$stash_ref, ['user', 'email_is_the_list', {email => $who}];
```
before the `next` line above and withdraw the changes below.

Reply:
ping @ldidry ?